Formula/fortio.rb

```diff
@@ -18,11 +18,8 @@ def install
 
     (buildpath/"src/istio.io/fortio").install buildpath.children
     cd "src/istio.io/fortio" do
-      date = Time.new.strftime("%Y-%m-%d %H:%M")
-      system "go", "build", "-a", "-o", bin/"fortio", "-ldflags",
-        "-s -X istio.io/fortio/ui.resourcesDir=#{lib} " \
-        "-X istio.io/fortio/version.tag=v#{version} " \
-        "-X \"istio.io/fortio/version.buildInfo=#{date}\""
+      system "make", "official-build", "GOOS=darwin",
+             "OFFICIAL_BIN=#{bin}/fortio", "LIB_DIR=#{lib}", "DATA_DIR=."
```

Comment:
Can you please explain why this should be `DATA_DIR=.` and not e.g. `DATA_DIR=#{var}/lib/istio/fortio`?

Reply:
because unless you run as a server using the current directory is most useful for data - only the docker image version uses var/lib as a volume

Reply:
@ilovezfs it also preserves the current behavior:
```
$ fortio 2>&1 | grep -A1 data-dir
  -data-dir string
    	Directory where JSON results are stored/read (default ".")
```
https://github.com/istio/fortio/blob/f1d27eafcee109fe6926b9741e3849de0ec636ce/fortio_main.go#L108

Reply:
Would be cool of `GOOS` could be autodetected in the `Makefile`. Otherwise 👍 thanks for creating it 👏

Reply:
I could have empty as the default and it would be native os, but in docker i build all 3 limux, macos, windows so I prefer to be explicit on which is built and passing it.

can we merge this ?

Reply:
I would need another release to change that detail. can we merge this first as this is clearly an improvement/what was asked ?

Reply:
Yup, no problem. (Also, one approach some take is to provide a specific Makefile target for building for the host OS such as `make dev` if you don't want that to be the behavior of the default target, or equivalently and more common, use a separate Makefile target for the docker builds that build for every OS.)

Reply:
fixing the GOOS part in #29532